proxies/nodejs/install.sh

```diff
@@ -1,4 +1,12 @@
-if [ -z "$NODEJS_SDK_VERSION" ]; then
+if [ "GITHUB_SHA" ]; then
+    git clone https://github.com/DevCycleHQ/js-sdks.git
+    cd js-sdks
+    git checkout $GITHUB_SHA
+    yarn
+    NX_DAEMON=false yarn nx build nodejs --verbose
+    cd ..
+    yarn add file:js-sdks/dist/sdk/nodejs/
```

Comment:
does this actually work without also adding all the dependent packages that are in this directory as well? Or is it installing those packages from NPM instead? Might want to make sure you're getting the local versions of all of them.

Reply:
looks like it's using those packages from NPM instead: ```    "@devcycle/bucketing-assembly-script": "^1.1.20",
    "@devcycle/types": "1.0.52",```

Reply:
you'll have to add each of those as well then